.storybook/preview.tsx

```diff
@@ -17,15 +17,15 @@ import { RequestContextProvider } from '../src/app/contexts/RequestContext';
 const REITH_SERIF_REGULAR = {
   '@font-face': {
     fontFamily: 'ReithSerif',
-    src: `url("fonts/Reith/BBCReithSerif_W_Rg.woff2") format("woff2"), url("fonts/Reith/BBCReithSerif_W_Rg.woff") format("woff")`,
+    src: "url('https://static.files.bbci.co.uk/fonts/reith/2.512/BBCReithSerif_W_Rg.woff2') format('woff2')",
```

Comment:
```suggestion
    src: "url('https://static.files.bbci.co.uk/fonts/reith/2.512/BBCReithSerif_W_Rg.woff2') format('woff2'), 'https://static.files.bbci.co.uk/fonts/reith/2.512/BBCReithSerif_W_Rg.woff') format('woff')",
```
I think we should be fetching the `woff` as well as `woff2` format fonts as we did previously, this is the case for the rest of the fonts too